addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2248,8 +2255,9 @@ var SnippetsMenu = Widget.extend({
                 // When noCheck is true, we only check the exclude.
                 return true;
             }
-            // `o_editable_media` bypasses the `o_not_editable` class.
-            if (this.classList.contains('o_editable_media')) {
+            // `o_editable_media` bypasses the `o_not_editable` class except for
+            // drag & drop.
+            if (!forDrop && this.classList.contains('o_editable_media')) {
```

Comment:
@qsm-odoo after looking at this a second time, it doesn't make sense to have a `o_editable_media` that is not in `o_not_editable` environment right ? I suppose we can get rid of this commit except if you want to support existing (wrong) custo that use the `o_editable_media` class in editable env..